kitsune/dashboards/templates/dashboards/contributors_old.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 {% from "dashboards/includes/macros.html" import print_readout, product_choice_list, print_subscription_menu with context %}
 {% from "dashboards/includes/macros.html" import localization_sidebar_nav %}
-{% set title = _('[{product}] Knowledge Base Dashboard')|f(product=_(product.title, 'DB: products.Product.title') if product else _('All Products')) %}
+{% set title = _('[{product}] Knowledge Base Dashboard')|f(product=pgettext('DB: products.Product.title', product.title) if product else _('All Products')) %}
```

Comment:
Using `|f` won't escape the product title. Seems like this also should get converted to `|fe`.